tests/unit/net/http_loader.rs

```diff
@@ -401,32 +394,41 @@ fn test_check_default_headers_loaded_in_every_request() {
     let mut en_us: LanguageTag = Default::default();
     en_us.language = Some("en".to_owned());
     en_us.region = Some("US".to_owned());
-    let mut en: LanguageTag = Default::default();
-    en.language = Some("en".to_owned());
```

Comment:
We should modify https://dxr.mozilla.org/servo/source/components/net/fetch/methods.rs#124 to call https://dxr.mozilla.org/servo/source/components/net/http_loader.rs#416 instead.